rotkehlchen/accounting/constants.py

```diff
@@ -1,206 +1,264 @@
+        HistoryEventSubType.NONE: EventCategory.RECEIVE,  # create for basic decoded events
     },
-    HistoryEventType.INFORMATIONAL: {
-        HistoryEventSubType.APPROVE: EventCategory.APPROVAL,
-        HistoryEventSubType.GOVERNANCE: EventCategory.GOVERNANCE,
-        HistoryEventSubType.DEPLOY: EventCategory.DEPLOY,
-        HistoryEventSubType.REMOVE_ASSET: EventCategory.WITHDRAW,
-        HistoryEventSubType.PLACE_ORDER: EventCategory.PLACE_ORDER,
-        HistoryEventSubType.NONE: EventCategory.INFORMATIONAL,
-    },
-    HistoryEventType.TRANSFER: {
-        HistoryEventSubType.NONE: EventCategory.TRANSFER,
+    HistoryEventType.DEPOSIT: {
+        HistoryEventSubType.DEPOSIT_ASSET: EventCategory.DEPOSIT,
+        HistoryEventSubType.NONE: EventCategory.DEPOSIT,
+        HistoryEventSubType.BRIDGE: EventCategory.BRIDGE_IN,
+        HistoryEventSubType.PLACE_ORDER: EventCategory.DEPOSIT,
+        HistoryEventSubType.RECEIVE_WRAPPED: EventCategory.RECEIVE,
+        HistoryEventSubType.FEE: EventCategory.FEE,  # verify
     },
-    HistoryEventType.TRADE: {
-        HistoryEventSubType.SPEND: EventCategory.SWAP_OUT,
-        HistoryEventSubType.RECEIVE: EventCategory.SWAP_IN,
+    HistoryEventType.SPEND: {
+        HistoryEventSubType.RETURN_WRAPPED: EventCategory.SEND,
+        HistoryEventSubType.LIQUIDATE: EventCategory.LIQUIDATE,
+        HistoryEventSubType.PAYBACK_DEBT: EventCategory.REPAY,
         HistoryEventSubType.FEE: EventCategory.FEE,
+        HistoryEventSubType.DONATE: EventCategory.DONATE,
+        HistoryEventSubType.DEPOSIT_ASSET: EventCategory.DEPOSIT,
+        HistoryEventSubType.NONE: EventCategory.SEND,  # create for basic decoded events
     },
     HistoryEventType.WITHDRAWAL: {
-        HistoryEventSubType.NONE: EventCategory.WITHDRAW,
         HistoryEventSubType.REMOVE_ASSET: EventCategory.WITHDRAW,
-        HistoryEventSubType.GENERATE_DEBT: EventCategory.BORROW,
+        HistoryEventSubType.NONE: EventCategory.WITHDRAW,
         HistoryEventSubType.BRIDGE: EventCategory.BRIDGE_OUT,
+        HistoryEventSubType.CANCEL_ORDER: EventCategory.CANCEL_ORDER,
+        HistoryEventSubType.REFUND: EventCategory.REFUND,
+        HistoryEventSubType.GENERATE_DEBT: EventCategory.BORROW,
+        HistoryEventSubType.RETURN_WRAPPED: EventCategory.SEND,
+        HistoryEventSubType.FEE: EventCategory.FEE,
+        HistoryEventSubType.REWARD: EventCategory.CLAIM_REWARD,  # verify
     },
-    HistoryEventType.DEPOSIT: {
-        HistoryEventSubType.NONE: EventCategory.DEPOSIT,
-        HistoryEventSubType.DEPOSIT_ASSET: EventCategory.DEPOSIT,
-        HistoryEventSubType.BRIDGE: EventCategory.BRIDGE_IN,
-        HistoryEventSubType.PLACE_ORDER: EventCategory.PLACE_ORDER,
+    HistoryEventType.TRADE: {
+        HistoryEventSubType.SPEND: EventCategory.SWAP_OUT,
+        HistoryEventSubType.RECEIVE: EventCategory.SWAP_IN,
+        HistoryEventSubType.NONE: EventCategory.INFORMATIONAL,  # verify
```

Comment:
That was hard to grep for. But I can't find it. Also it makes no sense. Trade/None does not make sense as a type. I don't know where you saw this. I would remove.

```suggestion
```

Reply:
These are used in tests: https://github.com/rotki/rotki/blob/develop/rotkehlchen/tests/db/test_history_events.py#L31
Should be changed to SPEND/RECEIVE accordingly I guess.